pkg/porter/porter.go

```diff
@@ -100,10 +99,6 @@ func (p *Porter) Connect(ctx context.Context) error {
 
 // Close releases resources used by Porter before terminating the application.
 func (p *Porter) Close() error {
-	if p.Debug {
```

Comment:
Do we not want to print this to debug level log anymore?

Reply:
Yeah it's not a helpful debug statement and would require that I start passing context.Context to Close which you had previously encouraged me to not do. 😀